dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorArm8.java

```diff
@@ -37,7 +37,14 @@
 
 class VisitorArm8 extends VisitorBase implements EventVisitor<List<Event>> {
 
-	protected VisitorArm8() {}
+	// If the source WMM does not allow OOTA behaviors (e.g. RC11)
+	// we need to strength the compilation following the paper
+	// "Outlawing Ghosts: Avoiding Out-of-Thin-Air Results"
+	private final boolean nooota; 
```

Comment:
Again, this is very misleading. "rc11Compilation" or "strongCompilation" are better fits for this.